packages/venia-concept/src/sw.js

```diff
@@ -0,0 +1,34 @@
+workbox.skipWaiting();
+workbox.clientsClaim();
+
+workbox.routing.registerRoute('/', workbox.strategies.staleWhileRevalidate());
+
+workbox.routing.registerRoute(
+    new RegExp('\\.html$'),
+    workbox.strategies.networkFirst()
+);
+
+workbox.routing.registerRoute(
+    new RegExp('/.\\.js$'),
+    workbox.strategies.staleWhileRevalidate()
+);
+
+workbox.routing.registerRoute(
```

Comment:
This works for static files, but catalog images should probably use `staleWhileRevalidate`. It would be this route, prepended with `/media/catalog`.